src/pages/en/guides/imports.md

```diff
@@ -17,9 +17,37 @@ Astro uses Vite as its internal build system. Vite provides Astro with built-in
 - Markdown (`.md`)
 - WASM (`.wasm`)
 
-Any files in your `public/` directory are copied into the final build, untouched by Vite or Astro. The following applies to files in your `src/` directory, which Astro is ultimately responsible for.
+## `public/`
+Any files in your `public/` directory are copied into the final build, untouched by Vite or Astro. 
 
-## JavaScript & ESM
+Reference them in your Astro component template via absolute URL paths, from the `public/` directory.
+
+```
+├── public/
+│   ├── images/
+|   |   └── photo1.jpg
+│   ├── styles/
+|   |   └── global.css
+|   └── utils.js
+```
+
+```astro
+---
+// Astro component script
+---
+<head><link rel="stylesheet" href="/styles/global.css"></head>
+<body>
+  <img src="/images/photo1.jpg"> 
+
+  <script type="module" src="/nav-utils.js"></script>
+</body>
+
+```
+
+## `src/`
+ The following applies to files in your `src/` directory, which Astro is ultimately responsible for.
+
+### JavaScript & ESM
 
 Astro was designed for JavaScript’s native ES Module (ESM) syntax. ESM lets you define explicit imports & exports that browsers and build tools can better understand and optimize for. If you're familiar with the `import` and `export` keywords in JavaScript, then you already know ESM!
```

Comment:
```suggestion
Astro was designed for JavaScript’s native ES Module (ESM) syntax. ESM lets you define explicit _imports_ & _exports_ that browsers and build tools can better understand and optimize for. If you're familiar with the `import` and `export` keywords in JavaScript, then you already know ESM!
```

Reply:
I don't think this helps the paragraph since import and export are inlined later. Italics isn't the *best* for accessibility (bold is better) and I'm not sure I'd do either one here, since I think "regular words" suit this description, followed up by the inline emphasis.